docker-compose.yaml

```diff
@@ -1,24 +1,24 @@
 ---
 services:
   unifi:
-    image: 'jacobalberty/unifi:${UNIFI_VERSION:?}'
```

Comment:
Ha, personally I prefer single quotes but I am okay either way as long as it's consistent. I have been meaning to setup `yamllint`

Reply:
Yeah I just have `prettier` installed for work and got tired of fighting it 😭